### PR TITLE
Use katip for logging in the executor

### PR DESCRIPTION
**Pull request by nc6**
Fairly minimal at the moment, but puts the framework in place.

### CODE REVIEW

File: src/Control/FunFlow/ContentStore.hs [hunk 1]
```diff
@@ -72,16 +72,18 @@ import           Prelude                         hiding (lookup)
 import           Control.Concurrent              (threadDelay)
 import           Control.Concurrent.Async
 import           Control.Concurrent.MVar
-import           Control.Exception               (Exception, bracket, bracket_,
+import           Control.Exception               (Exception, bracket_,
                                                   catch, throwIO)
 import           Control.Monad                   (filterM, forever, void)
+import Control.Monad.Catch (MonadMask, bracket)
```
Comment (aherrmann) [inline src/Control/FunFlow/ContentStore.hs]: formatting

File: src/Control/FunFlow/Exec/RedisJobs.hs [hunk 2]
```diff
@@ -14,6 +14,7 @@ module Control.FunFlow.Exec.RedisJobs where
 
 import           Control.FunFlow.Exec.Redis
 import           Control.FunFlow.External.Coordinator.Redis
+import           Control.Lens hiding (argument)
```
Comment (aherrmann) [inline src/Control/FunFlow/Exec/RedisJobs.hs]: formatting

File: src/Control/FunFlow/External/Executor.hs [hunk 3]
```diff
@@ -35,10 +40,10 @@ data ExecutionResult =
     --   TODO where should logs go?
   | Failure TimeSpec Int
 
--- | Execute an individual task.
-execute :: CS.ContentStore -> TaskDescription -> IO ExecutionResult
+  -- | Execute an individual task.
```
Comment (aherrmann) [inline src/Control/FunFlow/External/Executor.hs]: why indent?

Comment (nc6) [reply src/Control/FunFlow/External/Executor.hs]: Ugh, don't know, clearly my forays into emacs aren't entirely going well!

**Pull request merged**